openwisp_controller/config/tests/test_template.py

```diff
@@ -606,6 +606,7 @@ def test_config_modified_signal(self):
         # refresh instance to reset _just_created attribute
         conf = Config.objects.get(pk=conf.pk)
 
+        conf._delete_config_modified_timeout_cache()
```

Comment:
This reminds me of the django-allauth rate limiting which we had to disable in tests.

The problem with this approach of modifying each test is that all contributors have to be aware of this, which is not scalable for future maintenance.

We need to look for a solution that doesn't require contributors to be aware of this all the time.

What if we make the timeout configurable with a setting? That would allow us to disable it globally during tests and enable it only in the subset of tests where it is needed.